sentry-anyhow/src/lib.rs

```diff
@@ -1,4 +1,17 @@
-//! Adds support for capturing Sentry errors from `anyhow::Error`.
+//! Adds support for capturing Sentry errors from [`anyhow::Error`].
+//!
+//! This integration adds a new event *source*, which allows you to create events directly
+//! from an [`anyhow::Error`] struct.  As it is only an event source it only needs to be
+//! enabled using the `anyhow` cargo feature, it does not need to be enabled in the call to
+//! [`sentry::init`](../../fn.init.html).
```

Comment:
I guess these links are made in a way that they work when you are building `sentry` docs, and you expose this via `integrations::anyhow`. However you should still be able to build the docs for this individually. Maybe intra-doc-links will just work when you have `sentry` as a (circular) dev-dependency?

Reply:
ugh, this was the only solution I could get this to work reasonably.  I didn't think of it being standalone.  I guess that's how it ends up on docs.rs though?

I tried adding the circular dependency under the "doc" feature but cargo was not having any of that.  Do you think as a dev dependency would have more chance of working?

Reply:
oh, sentry was already a circular dev dependency.  so that doesn't work.  So it's basically between this option which break the doc.rs build for the standalone crate, or point it to `https://docs.rs/sentry/*/sentry/fn.init.html` I think.  Maybe the latter is the least evil :(